crates/node_binding/binding.js

```diff
@@ -8,18 +8,8 @@ let localFileExisted = false
 let loadError = null
 
 function isMusl() {
-  // For Node 10
-  if (!process.report || typeof process.report.getReport !== 'function') {
-    try {
-      const lddPath = require('child_process').execSync('which ldd').toString().trim();
-      return readFileSync(lddPath, 'utf8').includes('musl')
-    } catch (e) {
-      return true
-    }
-  } else {
-    const { glibcVersionRuntime } = process.report.getReport().header
-    return !glibcVersionRuntime
-  }
+  const { glibcVersionRuntime } = process.report.getReport().header
```

Comment:
getReport has performance issue, we should fix it using same things like https://github.com/napi-rs/napi-rs/pull/2531